src/net.cpp

```diff
@@ -1269,6 +1269,7 @@ void CConnman::NotifyNumConnectionsChanged()
     }
 }
 
+#ifdef WIN32
```

Comment:
IMO you could include this [m4](https://www.gnu.org/software/autoconf-archive/ax_have_poll.html) to detect if the system has poll.